src/x/ident/types.go

```diff
@@ -310,3 +310,11 @@ func (t Tags) Equal(other Tags) bool {
 	}
 	return true
 }
+
+// IndexChecksumBlock represents a set of index checksums within a series block.
+type IndexChecksumBlock struct {
+	// Checksums is the list of index checksums.
+	Checksums []int64
+	// Marker is a batch marker, signifying the ID of the last element in the batch.
+	Marker []byte
```

Comment:
Nit: more descriptive name, like `EndMarker` maybe (as `Marker` can be a marker for anything)?